public/app/plugins/datasource/jaeger/CheatSheet.tsx

```diff
@@ -0,0 +1,58 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { useStyles2 } from '@grafana/ui';
+
+const linkColor = '#1F62E0';
```

Comment:
This color exists in palette.ts - so I mean that we can use the color via our [theme](https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/themes/createColors.ts#L177) - that way it updates depending on whether we have light or dark mode selected.

So essentially, import GrafanaTheme2, add it to your styles and use the linked property above to get the color.